client/src/app/pages/applications/applications-table/applications-table.tsx

```diff
@@ -575,6 +574,23 @@ export const ApplicationsTable: React.FC = () => {
         >
           {t("actions.delete")}
         </DropdownItem>,
+        ...(tasksReadAccess && tasksWriteAccess
+          ? [
+              <DropdownItem
+                key="applications-bulk-cancel"
+                isDisabled={
+                  !selectedRows.some((application: DecoratedApplication) =>
+                    isTaskCancellable(application)
+                  )
+                }
+                onClick={() => {
+                  handleCancelBulkAnalysis();
```

Comment:
We need a confirmation action here - similar to deleting apps - search for references to `applicationsToDelete`.